src/components/organisms/UiDatepicker/UiDatepicker.vue

```diff
@@ -614,6 +614,8 @@ provide<ComputedRef<Record<string, DatepickerDatePart>>>('inputsIds', inputsIds)
 
   &__label {
     display: block;
+    font: functions.var($element + '-label', font, var(--font-body-2-comfortable));
+    letter-spacing: functions.var($element + '-label', letter-spacing, var(--letter-spacing-2-comfortable));
```

Comment:
```suggestion
    font: functions.var($element + '-label', font, var(--font-body-2-comfortable));
    letter-spacing: functions.var($element + '-label', letter-spacing, var(--letter-spacing-2-comfortable));
```
You can use `ui-text--body-2-comfortable` modifier for UiText element

Reply:
```
<UiText
  v-bind="getTextAttrs(datePart)"
  class="ui-text--body-2-comfortable ui-datepicker__label"
>
```